vertical-pod-autoscaler/e2e/updater.go

```diff
@@ -78,18 +74,96 @@ var _ = UpdaterE2eDescribe("Updater", func() {
 				UpperBound:    resourceList,
 			}},
 		}
-
-		vpaClientSet := vpa_clientset.NewForConfigOrDie(config)
-		vpaClient := vpaClientSet.PocV1alpha1()
-		_, err = vpaClient.VerticalPodAutoscalers(ns).Create(vpaCRD)
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		InstallVPA(f, vpaCRD)
 
 		ginkgo.By("Waiting for pods to be restarted")
 
 		err = waitForPodSetChangedInDeployment(c, d, podList)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 	})
+
+	ginkgo.It("observes pod disruption budget", func() {
+
+		ginkgo.By("Setting up a hamster deployment")
+		c := f.ClientSet
+		ns := f.Namespace.Name
+
+		cpuQuantity := ParseQuantityOrDie("10m")
+		memoryQuantity := ParseQuantityOrDie("10Mi")
+
+		d := NewHamsterDeploymentWithResources(f, cpuQuantity, memoryQuantity)
+		replicas := int32(10)
+		d.Spec.Replicas = &replicas
+		d, err := c.AppsV1().Deployments(ns).Create(d)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		err = framework.WaitForDeploymentComplete(c, d)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		podList, err := framework.GetPodsForDeployment(c, d)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		podSet := makePodSet(podList)
+
+		ginkgo.By("Setting up prohibitive PDB for hamster deployment")
```

Comment:
If you extract this too (even though it's used only once), the test will become easier to read - there is no need to skim the code to next ginkgo.By then.